lighthouse-core/gather/gatherers/form-elements.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const pageFunctions = require('../../lib/page-functions.js');
+
+/* eslint-env browser, node */
+
+/**
+ * @return {LH.Artifacts['FormElements']}
+ */
+/* istanbul ignore next */
+function collectFormElements() {
+  // @ts-ignore - put into scope via stringification
```

Comment:
we updated all of our ts-ignores since the PR was opened, sorry for the merge updates :)

```suggestion
  // @ts-expect-error - put into scope via stringification
```